src/parsing/binast-serialize-visitor.h

```diff
@@ -84,7 +84,6 @@ inline void BinAstSerializeVisitor::SerializeUint8(uint8_t value) {
 
 void BinAstSerializeVisitor::SerializeRawString(const AstRawString* s) {
   DCHECK(s != nullptr);
-  DCHECK(string_table_indices_.count(s) == 0);
```

Comment:
was this incorrect? since insert is called on this in loops SerializeStringTable, it seems guaranteed to be nonzero at some point. was encountering this issue when running test_binast

Reply:
It's testing for the presence of the string in the table. We should only ever serialize an AstRawString once. This assert firing means that there's something wrong with that assumption, so yeah you'll need to debug why this is happening b/c it shouldn't be.

Reply:
My first guess would be that we're reusing the same BinAstSerializeVisitor in multiple contexts, i.e. the string_table_indices were already filled in by something and then we tried to add the same basic constant strings again.

Reply:
Turns out my guess was wrong :sweat_smile: